stripper/ze_zertinan_v1_4.cfg

```diff
@@ -0,0 +1,92 @@
+// fix entwatch not tracking some item cooldowns
+modify:
+{
+	match:
+	{
+		"targetname" "Item_Fire_Button"
+		"classname" "func_button"
+	}
+	replace:
+	{
+		"parentname" "Item_Fire_elite"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"targetname" "Item_Ice_Button"
+		"classname" "func_button"
+	}
+	replace:
+	{
+		"parentname" "Item_Ice_elite"
+	}
+}
```

Comment:
This isn't necessary, `buttonid` was added to entWatch specifically so we would avoid needing to do this.